### ISSUE TITLE
ICCV 2023 | A Unified Continual Learning Framework with General Parameter-Efficient Tuning

### ISSUE THREAD

**Issue by ChufanSuki** [ChufanSuki/read-paper-and-code#86]
https://arxiv.org/abs/2303.10070

https://github.com/gqk/lae

**Comment by ChufanSuki** [comment]
### Motivation

Setting: Pre-train $\rightarrow$ downstream adaptation in continual learning 

Existing methods: L2P, DualPrompt, ESN

Limitations: 

1. They are all constrained to a specific PET method $\rightarrow$ transfomer only
2. Most of them rely on selecting task-specific parameter for each individual task $\rightarrow$ The selection tends to be noisy with increasing task numbers  and the task-specific prompts appear homogeneous

**Comment by ChufanSuki** [comment]
model $f(\cdot;\theta, \phi)$

Pre-trained feature extractors: $\theta_{pre}$

Classifier $\phi_{old}$ for all learned tasks $\mathcal{T}_{1:i}$

Classifier $\phi_{new}$ for current learning task $\mathcal{T}_{i}$


Naive Baseline: Seq-FT

### Proposed Framework: LAE

### Learning with calibrated speed

At the beginning of the training, we only learn $\phi_{new}$ with $\theta_{pet}$ frozen; 

Then after $\phi_{new}$ has sufficiently learned and the loss has significantly decreased, we jointly learn both $\phi_{new}$ and $\theta_{pet}$.

#### Accumulation of multi-task knowledge
